ui/app/AppLayouts/Chat/popups/GroupInfoPopup.qml

```diff
@@ -285,154 +193,106 @@ ModalPopup {
-                        }
-                    }
-                }
-
-                StyledText {
-                    id: adminLabel
-                    visible: model.isAdmin
-                    //% "Admin"
-                    text: qsTrId("group-chat-admin")
-                    anchors.right: parent.right
-                    anchors.verticalCenter: parent.verticalCenter
-                    font.pixelSize: 13
-                    color: Style.current.secondaryText
-                }
-
-                StyledText {
-                    id: moreActionsBtn
-                    visible: !model.isAdmin && popup.isAdmin
-                    text: "..."
-                    anchors.right: parent.right
-                    anchors.rightMargin: Style.current.smallPadding
-                    anchors.verticalCenter: parent.verticalCenter
-                    font.pixelSize: 20
-                    font.bold: true
-                    color: Style.current.secondaryText
-                    MouseArea {
-                        anchors.fill: parent
+                title: !model.userName.endsWith(".eth") && !!contactRow.nickname ?
+                           contactRow.nickname : Utils.removeStatusEns(model.userName)
+                //% "(You)"
+                titleAsideText: model.publicKey === userProfile.pubKey ? qsTrId("-you-") : ""
+                //% "Admin"
+                statusListItemTitle.font.pixelSize: 17
+                statusListItemTitleAside.font.pixelSize: 17
+                label: model.isAdmin ? qsTrId("group-chat-admin"): ""
+                image.source: appMain.getProfileImage(model.publicKey)|| model.identicon
+                image.isIdenticon: true
+                components: [
+                    StatusFlatRoundButton {
+                        id: moreActionsBtn
+                        type: StatusFlatRoundButton.Type.Tertiary
+                        color: "transparent"
+                        icon.name: "more"
+                        icon.color: Theme.palette.baseColor1
+                        visible: !model.isAdmin && popup.isAdmin
                         onClicked: {
                             contextMenu.popup(-contextMenu.width / 2 + moreActionsBtn.width / 2, moreActionsBtn.height + 10)
                         }
-                        cursorShape: Qt.PointingHandCursor
-                        // TODO: replace with StatusPopupMenu
-                        PopupMenu {
+                        StatusPopupMenu {
                             id: contextMenu
-                            Action {
-                                icon.source: Style.svg("make-admin")
-                                icon.width: 16
-                                icon.height: 16
+                            StatusMenuItem {
+                                image.source: Style.svg("make-admin")
```

Comment:
This should be using `icon` or `iconSettings` from `StatusMenuItem` (and therefore use icons provided by StatusQ as well)